CONTRIBUTING.md

```diff
@@ -35,7 +35,7 @@ This is an example to run only Jarbas. First run migrations and provision:
 
 ```console
 $ docker-compose run --rm django python manage.py migrate
-$ docker-compose run --rm django python manage.py reimbursements /mnt/data/reimbursements_sample.xz
+$ docker-compose run --rm django python manage.py reimbursements /mnt/data/reimbursements-2018.csv
```

Comment:
The `reimbursement_sample.xz` is committed as a way to make it easy to get a functional version of Jarbas. Are we committing a new sample named  `reimbursements-2018.csv`? I don't see it in this PR so I'm not sure what we're attempting here.